test/modules/geo-layers/mvt-layer.spec.js

```diff
@@ -180,15 +191,16 @@ test('MVT Highlight', async t => {
         filled: true,
         pickable: true,
         autoHighlight: true,
-        highlightedFeatureId: 1
+        highlightedFeatureId: 1,
+        binary: false
       },
-      onAfterUpdate: ({subLayers}) => {
-        for (const layer of subLayers) {
-          t.ok(layer.props.pickable, 'MVT Sublayer is pickable');
-          t.ok(!layer.props.autoHighlight, 'AutoHighlight should be disabled');
-          t.equal(layer.props.highlightedObjectIndex, 0, 'Feature highlighted has index 0');
-        }
-      }
+      onAfterUpdate
+    },
+    {
+      updateProps: {
+        binary: true
```

Comment:
What does this test? I don't see the layer re-fetch tile data when the `binary` prop changes.

Reply:
You're totally right it was not called

Reply:
It tests highlight with binary data